user_tools/src/spark_rapids_pytools/resources/qualification-conf.yaml

```diff
@@ -4,6 +4,7 @@ toolOutput:
   textFormat:
     summaryLog:
       fileName: rapids_4_spark_qualification_output.log
+    log4jFileName: rapids_4_spark_qualification_stderr.log
```

Comment:
QQ: Double checking that it is verified that the content of this file does not get included in the CI/CD tests that compare the output of 2 runs.